spharpy/spherical.py

```diff
@@ -130,21 +490,180 @@ def spherical_harmonic_basis(n_max, coords):
 
     """
 
-    n_coeff = (n_max+1)**2
+    n_coeff = (n_max + 1) ** 2
 
     basis = np.zeros((coords.csize, n_coeff), dtype=complex)
 
     for acn in range(n_coeff):
-        order, degree = acn2nm(acn)
+        if channel_convention == 'fuma':
+            order, degree = fuma2nm(acn)
+        else:
+            order, degree = acn2nm(acn)
         basis[:, acn] = _special.spherical_harmonic(
             order,
             degree,
             coords.colatitude,
             coords.azimuth)
+        if normalization == 'sn3d':
+            basis[:, acn] *= to_sn3d_norm(degree, order)
+        elif normalization == 'maxN':
+            basis[:, acn] *= to_maxN_norm(acn)
+
+    return basis
+
+
+def spherical_harmonic_basis_real(n_max, coords, normalization='n3d', channel_convention='acn'):
+    r"""
+    Calculates the real valued spherical harmonic basis matrix.
+
+    The spherical harmonic functions are fully normalized (N3D) and follow
```

Comment:
this doctoring is not valid when the user puts sn3d as a normalization or am I wrong?